cobspec/src/test/java/com/saralein/cobspec/data/CookieStoreTest.java

```diff
@@ -5,32 +5,25 @@
 import org.junit.Test;
 import java.util.ArrayList;
 import java.util.List;
-import java.util.stream.Collectors;
 import static org.junit.Assert.*;
 
 public class CookieStoreTest {
-    private final List<Cookie> TWO_COOKIE_LIST = new ArrayList<Cookie>() {{
-        add(new Cookie("type", "chocolate"));
-        add(new Cookie("baker", "Phil"));
-    }};
+    private List<Cookie> twoCookies;
     private CookieStore cookieStore;
 
+    private boolean cookiesAreEqual(List<Cookie> expected, List<Cookie> actual) {
+        expected.sort(Cookie::compareTo);
+        actual.sort(Cookie::compareTo);
+        return expected.equals(actual);
+    }
```

Comment:
If your `Cookie` class is implementing the `hashCode` function, you could compare the collections of cookies by converting them to a `Set`

```java
public void assertCookiesAreEqual(List<Cookie> expected, List<Cookie> actual) {
    assertEquals(new HashSet<Cookie>(expected), new HashSet<Cookie>(actual));
}
```

(That may not be exactly right, but I think it is close.)

Reply:
Thank you.  I had hashCode at one point and was using it for something, but didn't realize I could use it like this.

Right now I just get the hash code for the name and value and add them together.  I did read a hash code recommendation that was something like this:

```
    public int hashCode() {
        int result = 17;
        result = 31 * result + name.hashCode();
        result = 31 * result + value.hashCode();
        return result;
    }
```

Do you all do something like this when you override hashCode?

Reply:
Yeah, I always let Intellij do it for me 😄

Reply:
And the code it generates always looks something similar to what you have there 👆

Reply:
Wow.  mind = 💥  I wouldn't have thought it could do that and I need to share it with everyone I talk to for the rest of the day.  Going to update my code with its version.

Reply:
Yeah, it is really useful! But I only recommend that someone uses it once they understand what it is doing and why they need `hashCode`. 

Intellij can also generate `equals` and `toString` for you, fyi.